Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -129,6 +127,28 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate {
             }
         }
     }
+
```

Comment:
The reader button is part of BrowserLocationView, so wouldn't that interfere with it?